pom.xml

```diff
@@ -129,25 +131,24 @@
         <dependency>
             <groupId>org.apache.jackrabbit</groupId>
             <artifactId>oak-core</artifactId>
-            <version>1.8.8</version>
+            <version>${oak.version}</version>
             <scope>provided</scope>
         </dependency>
         <dependency>
             <groupId>org.apache.jackrabbit</groupId>
             <artifactId>oak-jcr</artifactId>
-            <version>1.8.8</version>
+            <version>${oak.version}</version>
             <scope>provided</scope>
         </dependency>
         <dependency>
             <groupId>org.apache.sling</groupId>
             <artifactId>org.apache.sling.api</artifactId>
-            <version>2.18.4</version>
+            <version>2.24.0</version>
```

Comment:
What makes changing the minimum version of this dependency necessary?

Reply:
Actually, no dependency version is necessary, but just keeping them to the latest versions.

Reply:
Ok, just be aware that changing this has altered the Import-Package instruction in the manifest for the used packages from this artifact.

For example:
_org.apache.sling.api.resource;version="[2.11,3)"_ has changed to: _org.apache.sling.api.resource;version="[2.13,3)"_

In general, if it still works with the older version and there are no security bugs or other compelling reasons for switching to the new version then it may be better to remain depending on the older version.

Reply:
Ok, make sense since its working fine and compatible with the older version, I reverted back to normal.